### PR TITLE
Work on cross compilation

### PR DESCRIPTION
**Pull request by dlinov**

### CODE REVIEW

File: frontend/src/main/scala/com/github/nikalaikina/tetris/http/GameApp.scala [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+
+    // it's a CancelableFuture, so there is a way to stop/restart the game
+    merged.runAsyncGetLast
+  }
+
+  def drawMatrix(ctx: dom.CanvasRenderingContext2D, matrix: Matrix, width: Int, height: Int): Unit = {
+    ctx.fillStyle = Color.Yellow
+    ctx.fillRect(0, 0, width, height)
+    ctx.strokeStyle = Color.Black
+    matrix.show.foldLeft(0) { (i, row) ⇒
```
Comment (nikalaikina) [inline frontend/src/main/scala/com/github/nikalaikina/tetris/http/GameApp.scala]: Why foldLeft and not zipWithIndex + for?

File: shared/src/main/scala/com/github/nikalaikina/tetris/Action.scala [hunk 2]
```diff
@@ -0,0 +1,7 @@
+package com.github.nikalaikina.tetris
+
+sealed trait Action
```
Comment (nikalaikina) [inline shared/src/main/scala/com/github/nikalaikina/tetris/Action.scala]: I don't really love this 3-lines-of-code files

File: cli/src/main/scala/com/github/nikalaikina/tetris/cli/Boot.scala [hunk 3]
```diff
@@ -1,6 +1,7 @@
-package com.github.nikalaikina.tetris
+package com.github.nikalaikina.tetris.cli
```
Comment (nikalaikina) [inline cli/src/main/scala/com/github/nikalaikina/tetris/cli/Boot.scala]: You can remove cli version

**Review by nikalaikina — APPROVED** [review]

File: frontend/src/main/scala/com/github/nikalaikina/tetris/http/GameApp.scala (hunk 1 above)
Comment (dlinov) [reply frontend/src/main/scala/com/github/nikalaikina/tetris/http/GameApp.scala]: `zipWithIndex` iterates over whole collection and creates additional intermediate collection

Comment (nikalaikina) [reply frontend/src/main/scala/com/github/nikalaikina/tetris/http/GameApp.scala]: I'm ok with that. Shouldn't I?

Comment (dlinov) [reply frontend/src/main/scala/com/github/nikalaikina/tetris/http/GameApp.scala]: 👌

**Pull request merged**